python/rikai/torch/transforms.py

```diff
@@ -12,17 +12,18 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-
-from typing import Mapping
+from typing import Any, Mapping
 
 import numpy as np
 import pandas as pd
 
-from rikai.mixin import ToNumpy
+from rikai.mixin import ToNumpy, ToPIL
+
+__all__ = ["RikaiToTensor"]
 
 
-def convert_tensor(row):
-    """Convert row into rikai semantic objects."""
+def convert_tensor(row, use_pil: bool = False):
```

Comment:
nit: the extra boolean parameter is a bit confusing because it took some tracing to figure out where it was being set to True etc.

Reply:
What would be an appropriate way to write parameter here? `use_pil = None` or making it a required parameter?